Basic/Basic/MainViewController.m

```diff
@@ -57,11 +68,37 @@ - (void) prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
     }
     else if ([segue.identifier isEqualToString:@"toCreateEmployee"])
     {
+        NSEntityDescription *decs = [NSEntityDescription entityForName:@"EmployeeModel" inManagedObjectContext:self.managedObjectContext];
         CreateEmployeeViewController *createEVC = (CreateEmployeeViewController *)segue.destinationViewController;
         createEVC.delagate = self;
+        createEVC.employee = [[Employee alloc] initWithEntity:decs insertIntoManagedObjectContext:self.managedObjectContext];
     }
 }
 
+- (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    return YES;
+}
+
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    if (editingStyle == UITableViewCellEditingStyleDelete)
+    {
+        Employee *ob = self.organization.employees.allObjects[(int)indexPath.row];
+        [self.organization removeEmployee: ob];
```

Comment:
Setup cascade delete rule relationship and you wouldn't have to remove objects directly.